src/test/java/com/wikia/webdriver/elements/fandom/components/LoadMore.java

```diff
@@ -0,0 +1,35 @@
+package com.wikia.webdriver.elements.fandom.components;
+
+import com.gargoylesoftware.htmlunit.ElementNotFoundException;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+
+/**
+ * Created by liz_lux on 1/26/17.
+ */
+public class LoadMore extends BasePageObject {
+
+    @FindBy(css = ".more-stories .load-more-btn")
+    private WebElement loadMoreButton;
+
+    public boolean isButtonDisplayed() {
+        try {
+            return loadMoreButton.isDisplayed();
+        } catch (ElementNotFoundException e) {
```

Comment:
The button is already on the page so we don't have to wait for it here. I updated the name of the other function though.